.github/workflows/test.yml

```diff
@@ -17,40 +17,69 @@ jobs:
     steps:
     - name: Build and run tests
       id: action-ros-ci
-      uses: ros-tooling/action-ros-ci@v0.2
+      uses: ros-tooling/action-ros-ci@v0.3
       with:
-        package-name: |
-          mcap_vendor
-          ros2bag
-          rosbag2
-          rosbag2_compression
-          rosbag2_compression_zstd
-          rosbag2_cpp
-          rosbag2_examples_cpp
-          rosbag2_interfaces
-          rosbag2_py
-          rosbag2_storage
-          rosbag2_storage_mcap
-          rosbag2_storage_sqlite3
-          rosbag2_test_common
-          rosbag2_test_msgdefs
-          rosbag2_tests
-          rosbag2_transport
-          shared_queues_vendor
-          sqlite3_vendor
-          zstd_vendor
         target-ros2-distro: rolling
         vcs-repo-file-url: https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos
         colcon-defaults: |
           {
             "build": {
               "cmake-args": [
-                "-DCMAKE_CXX_FLAGS=\"-Werror\""
+                "-DCMAKE_CXX_FLAGS=\"-Werror\"",
+                "-DBUILD_ROSBAG2_BENCHMARKS=1"
+              ],
+              "packages-up-to": [
+                "mcap_vendor",
+                "ros2bag",
+                "rosbag2",
+                "rosbag2_compression",
+                "rosbag2_compression_zstd",
+                "rosbag2_cpp",
+                "rosbag2_examples_cpp",
+                "rosbag2_examples_py",
+                "rosbag2_interfaces",
+                "rosbag2_performance_benchmarking",
+                "rosbag2_performance_benchmarking_msgs",
+                "rosbag2_py",
+                "rosbag2_storage",
+                "rosbag2_storage_mcap",
+                "rosbag2_storage_mcap_testdata",
```

Comment:
@MichaelOrlov `rosbag2_storage_mcap_testdata` doesn't exist, apparently:

> Package 'rosbag2_storage_mcap_testdata' specified with --packages-up-to was not found

https://github.com/ros2/rosbag2/actions/runs/4661838229/jobs/8251557954?pr=1268#step:3:7628

Reply:
Good catch. This is error from resolving conflicts during rebase `osbag2_storage_mcap_testdata` has been renamed to the `rosbag2_test_msgdefs`. I fixed it via fixup commit. Now we should be fine.